Gemfile.jruby-2.3.lock.release

```diff
@@ -271,7 +271,7 @@ GEM
     logstash-input-dead_letter_queue (1.1.1)
       logstash-codec-plain
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-input-elasticsearch (4.0.6)
+    logstash-input-elasticsearch (4.1.0)
```

Comment:
I'm OK with the change to add metadata into here

https://github.com/logstash-plugins/logstash-input-elasticsearch/blob/master/CHANGELOG.md